Alignment/CommonAlignment/plugins/FilterOutLowPt.cc

```diff
@@ -0,0 +1,183 @@
+// -*- C++ -*-
+//
+// Package:   Alignment/CommonAlignment
+// Class:     FilterOutLowPt
+//
+//
+// Original Author:  Marco Musich
+
+#include <memory>
+#include <vector>
+#include <map>
+#include <set>
+#include <utility>      // std::pair
+#include "TMath.h"
+
+// user include files
+
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+class FilterOutLowPt : public edm::stream::EDFilter<> {
+public:
+  explicit FilterOutLowPt( const edm::ParameterSet & );
+  ~FilterOutLowPt();
+  
+private:
+  virtual void beginJob() ;
+  virtual bool filter ( edm::Event &, const edm::EventSetup&); 
+  virtual void endJob() ;
+
+  bool applyfilter;
+  bool debugOn;
+  double thresh;
+  unsigned int numtrack;
+  double  ptmin;
+  edm::InputTag tracks_;
+  double trials;
+  double passes;
+  bool runControl_;
+  std::vector<unsigned int> runControlNumbers_;
```

Comment:
@mmusich I think you could make these configuration parameters const and initialize them with an initializer list